CHANGELOG.rst

```diff
@@ -15,7 +15,12 @@ Subsections for each version can be one of the following;
 - ``Security`` in case of vulnerabilities.
 
 Each individual change should have a link to the pull request after the description of the change.
+1.3.1 (unreleased)
```

Comment:
If we have added this version to the changelog feels like we want to update the version file to match this 1.3.1

Reply:
[oops](https://github.com/lvgig/tubular/pull/258/commits/c9daccfc3e2feb0bea9dac3e9f58d2c38ec58c7a)